deploy/klusterlet/chart/klusterlet/templates/agent-namespace.yaml

```diff
@@ -6,6 +6,13 @@ kind: Namespace
 metadata:
   annotations:
     workload.openshift.io/allowed: "management"
+  labels:
+    gwcp.guidewire.com/app-name: cluster-manager
+    gwcp.guidewire.com/created-by: pod-vulcan
+    gwcp.guidewire.com/dept: "275"
+    gwcp.guidewire.com/maintained-by: pod-vulcan
+    gwcp.guidewire.com/tenant-name: quadrant-shared
+    gwcp.guidewire.com/version: 0.16.0
```

Comment:
Please remove the labels from all these templates. These were added for testing.